.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why do we need any settings ? 

Would it be possible to have same settings as we have for other projects ? (Very sorry for not communicating it upfront, i did not realize)

See settings here:
https://github.com/opendata-mvcr/mission-control/blob/main/package.json
https://github.com/opendata-mvcr/ontoGrapher/blob/main/package.json
https://github.com/kbss-cvut/termit-ui

As I understood from other project everywhere we use only default configuration. Which we considered quite ok. By having same settings everywhere we can copy code between projects easier and people does not have to switch to different layout.

Reply:
Indeed I did not think about checking config in other projects. But I added these settings to improve readability. Without config:
![image](https://user-images.githubusercontent.com/57750426/151513335-ef015b46-91d3-4440-b275-3e7ebfed35b9.png)


With Config:
![image](https://user-images.githubusercontent.com/57750426/151513413-010cda95-3920-4329-9140-42d4086e218a.png)


But yes, I agree that it is simplier not to have any config to be more consistent across different projects.